.travis.yml

```diff
@@ -1,25 +1,33 @@
 language: python
+
 python:
   - "3.5"
   - "3.6"
   - "3.7"
   - "3.8"
+
 env:
   - DJANGO=2.2
   - DJANGO=3.0
+
 install:
 # command to install dependencies
   - "pip install coveralls"
   - "pip install pytz" # Needed for tests
   - pip install -q Django==$DJANGO
   - "pip install ."
 # command to run tests
+
 script:
+  - SAMPLE_APP=1 coverage run --branch --source=notifications manage.py test
   - coverage run --branch --source=notifications manage.py test
+
 matrix:
   exclude:
     - python: "3.5"
       env: DJANGO=3.0
+    - python: "3.5"
+      env: SAMPLE_APP=1 DJANGO=3.0
```

Comment:
I think these 2 lines are not needed, because SAMPLE_APP=1 is not part of the env

Reply:
I must have forgotten to remove them after updating method by which tests were run for sample app. As earlier separate machines where spinning up for running tests for sample app.